src/bikeracks/votes.py

```diff
@@ -31,16 +31,12 @@ def get_vote_data(rack_id, user_id):
 
 @votes.route('/get_vote_status', methods=['GET'])
 def get_vote_status():
-    if request.method != 'GET':
-        return "Bad Request", 400
-    
     # return true if user with user_id = user_id has voted for 
     # rack with rack_id=rack_id, false otherwise
 
-    rack_id = request.args.get('rack_id') or None
-    user_id = request.args.get('user_id') or None
+    rack_id = request.args.get('rack_id', None)
+    user_id = request.args.get('user_id', None)
```

Comment:
This returns str by default right? Even if not, you want string here so you can pass it to `db.execute` later, right? Empty string is fine since it won't match anything.  If it doesn't already behave this way then you can use `type=str`. In any case, drop the `None` since you don't actually want that.

Reply:
I dropped the default types for all of them and just specified their types.